lib/elixir/lib/function.ex

```diff
@@ -166,4 +166,27 @@ defmodule Function do
   @doc since: "1.7.0"
   @spec info(fun, item) :: {item, term} when item: information
   def info(fun, item), do: :erlang.fun_info(fun, item)
+
+  @doc """
+  Returns its input `value`, which can be given as an anonymous function
+  to transformation functions.
+
+  ## Examples
+
+      iex> Function.identity("Hello world!")
+      "Hello world!"
+
+      iex> 'abcdaabccc' |> Enum.sort() |> Enum.chunk_by(&Function.identity/1)
+      ['aaa', 'bb', 'cccc', 'd']
+
+      iex> Enum.group_by('abracadabra', &Function.identity/1)
+      %{97 => 'aaaaa', 98 => 'bb', 99 => 'c', 100 => 'd', 114 => 'rr'}
+
+      iex> Enum.map([1, 2, 3, 4], &Function.identity/1)
+      [1, 2, 3, 4]
+
+  """
+  @doc since: "1.10.0"
```

Comment:
Shouldn't it just be "1.10"?

Reply:
No, it is always fully specified, because something could be added on 1.10.2, for example. It is rare though.

Reply:
I agree, but the all references to x.0 should be replaced with x.0.0

Reply:
Unless that's not the way we are doing it

Reply:
I can't think of anywhere we used `since: "x.y"`.

Reply:
Yes, you are right. I think i got confused the the two zeroes in 10.0

Reply:
Yes, you are right. I think i got confused the the two zeroes in 10.0